### PR TITLE
Sort python runner safe builtins

### PR DESCRIPTION
**Pull request by zhongjiajie**
Sort it and add unittest to avid regression

## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Other

## Description

Sort it and add unittest to avid regression

### CODE REVIEW

**Comment by zhongjiajie** [comment]
@gabrieldutra @rauchy @rafawendel PTAL, I sorted the python runner builtins package and add unittest for it

**Review by arikfr — COMMENTED** [review]
Thank you for this, but I'm not sure if this is a behavior that needs testing?

Also, we're using Black to format the codebase and it uses double quotes.

**Comment by zhongjiajie** [comment]
I add test for avoiding regression, it could be more easy to check if adding package break the order or not.
> Black to format the codebase and it uses double quotes.
And CI green, strangely

**Comment by zhongjiajie** [comment]
I change to double quotes, could you please take a look? @arikfr @kravets-levko

File: tests/query_runner/test_python.py
Comment (susodapop) [inline tests/query_runner/test_python.py:9]:
```suggestion
        assert src == sorted(src), 'Python safe_builtins package not sorted.'
```

**Review by susodapop — CHANGES_REQUESTED** [review]
LGTM. One nit for grammar. Happy to merge right away after.

Thank you again!

**Comment by susodapop** [comment]
Thank you. CI is flaky for build / tests right now. I've checked the code out to run locally. Will merge once I've confirmed the tests pass.

**Comment by susodapop** [comment]
I manually verified all tests (unit, e2e) pass and the build runs fine locally.

Thanks again for this contribution!

**Comment by susodapop** [comment]
I cloned this PR so it would pass the CI build. It failed as a side-effect of being open so long. Oops.

Interesting side note: this happened in #5781, exactly 500 issues/pulls after this one was opened. If I could merge it at 11:11 UTC I would 😆

**Pull request closed without merging**
